index.js

```diff
@@ -2125,6 +2125,26 @@
     return Extend.methods.extend(extend_)(f);
   }
 
+  //# duplicate :: Extend w => w a -> w (w a)
+  //.
+  //. Adds one level of nesting to a comonadic structure.
```

Comment:
@davidchambers I'm not fond of this description. I chose it to mirror the description for `join`, but given the `duplicate([1, 2, 3])` example it could be confusing.

Reply:
I find writing these descriptions very challenging. I think your description is fine, and one of the roles of doctests is to succinctly demonstrate things which are not easily explained in prose.